### ISSUE TITLE
Accents in webspace name render unicode in fulltext

### ISSUE THREAD

**Issue by Jupi007** [sulu/sulu#6827]
| Q | A
| --- | ---
| Sulu Version | 2.5.1
| PHP Version | 8.1
| DB Version | MariaDB 10.7.3
| Browser Version | Firefox 105.0.1

# Actual Behavior

Using accents in the webspace name, render character unicode in fulltext:

![Capture d’écran du 2022-09-28 11-12-27](https://user-images.githubusercontent.com/36476595/192740319-8c6a6d95-eeaf-48e8-a6bd-c4132e4cddb1.png)
**_Sulu 2.5.1_**

# Expected Behavior

Accents should be rendered correctly:

![Capture d’écran du 2022-09-28 11-13-37](https://user-images.githubusercontent.com/36476595/192740337-526eb731-98aa-4358-b690-489c8f9c9a0e.png)
**_Sulu 2.5.0_**

# Steps to Reproduce

- Using Sulu 2.5.1 ;
- Add accents in a webspace name ;
- Connect to the admin, and go to "Webspaces" ;
- See the problem.

I also tested it with Chromium 105: same rendering problem.
I'm sure it is related to a change between Sulu 2.5.0 and 2.5.1, in the way that webspace name are serialized.

Thanks by advance :)

**Comment by alexander-schranz** [comment]
Thx for report this should not happen. I think @mamazu did change here something in https://github.com/sulu/sulu/pull/6714. I think here some flags for `json_encode` are not correct and need to be added.

**Comment by Jupi007** [comment]
You're right, it comes from this:

https://github.com/sulu/sulu/pull/6714/files#diff-f4fe3c2e77f45c284b1108287baa7cda92cd8518c448ec85b579e71eee4e7099L35

If I revert it, the problem disappear.

**Comment by alexander-schranz** [comment]
@Jupi007 we can not revert it as it solve another problem. We should just use the correct flags for `json_encode` to avoid the `\u00e9`. Fast research it can be achieved via the following flag: https://3v4l.org/aCd1K. Do you want to create a pull request for the `2.4` branch here.

**Comment by Jupi007** [comment]
> @Jupi007 we can not revert it as it solve another problem.

Sorry, I didn't mean that this should be reverted :D
Just, you are right it comes from here, as it disappear when I revert it.

> We should just use the correct flags for `json_encode` to avoid the `\u00e9`. Fast research it can be achieved via the following flag: https://3v4l.org/aCd1K.

I also was searching a solution and I found the same one as you.
Define this flag fix the problem:

```php
$webspace->setName({{ webspace.name|json_encode(constant('JSON_UNESCAPED_UNICODE'))|raw }});
```

> Do you want to create a pull request for the `2.4` branch here.

With pleasure!

**Comment by alexander-schranz** [comment]
fixed in #6828 and will be part of the next releases

**Comment by mamazu** [comment]
I don't think that Twig makes sense to use in the first place. All the work-arounds we are doing just to render a simple string.
